### ISSUE TITLE
feat: Linux support

### ISSUE THREAD

**Issue by cablehead** [cablehead/stacks#50]
Do you use Linux as your daily driver, and have a passion for clipboard managers? We'd love to hear from you!

See #49 for notes on what's needed.

**Comment by cablehead** [comment]
See #49 for notes on what's needed.

**Comment by Munchotaur** [comment]
I can't lay claim to a passion for clipboard managers. Will clipboard-manager-curious do for now? Not a developer but happy to test, comment etc.

**Comment by cablehead** [comment]
@Munchotaur that'd be amazing 🙏..

**Comment by cablehead** [comment]
a failing build has been added for Linux. This is going to happen!

**Comment by marvin-j97** [comment]
For my X11 Ubuntu, @ChurchTao 's clipboard-rs has been working well

However Wayland support is an issue:
![image](https://github.com/user-attachments/assets/d181d2a4-c2f6-45a7-a6ac-0e02ff900044)

**Comment by Munchotaur** [comment]
Yeah, display stuff on Linux is going through another "awkward phase" with the gradual transition to Wayland. Not everyone likes Wayland either so who knows if it'll ever be complete?

**Comment by cablehead** [comment]
This is a standalone example to be able to experiment with the clipboard on different platforms

https://github.com/cablehead/xs/tree/main/examples/x-macos-pasteboard/solid-ui

**Comment by marvin-j97** [comment]
There is https://github.com/YaLTeR/wl-clipboard-rs

**Comment by cablehead** [comment]
@marvin-j97 has a cli to output the x11 clipboard to stdout here: https://github.com/marvin-j97/x11-pasteboard
